configs/components/fesom/fesom-2.1.yaml

```
@@ -189,8 +188,8 @@ restart_out_in_work:
         fesom_bin_restart_info: fesom_bin_restart/*.info
         fesom_bin_restart: fesom_bin_restart/np${nproc}/*
         wiso_restart: fesom.${end_date!syear}.wiso.restart.nc
-        icb_restart: iceberg.restart #.${parent_date!syear}
-        icb_restart_ISM: iceberg.restart.ISM
+        icb_restart: iceberg.restart #_${start_date!syear} #.${parent_date!syear}
+        icb_restart_ISM: iceberg.restart.ISM #_${start_date!syear}
```

Remove these comments if it works
```suggestion
        icb_restart: iceberg.restart
        icb_restart_ISM: iceberg.restart.ISM
```